<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeterLink</class>
 <widget class="QDialog" name="MeterLink">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Modbus ID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Modbus IP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ModBus port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>equipmentID</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>404</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="11">
      <widget class="QLineEdit" name="lineEdit_21">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="20">
      <widget class="QLabel" name="label_41">
       <property name="text">
        <string>imp_kvah</string>
       </property>
      </widget>
     </item>
     <item row="2" column="16">
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>tot_kwh</string>
       </property>
      </widget>
     </item>
     <item row="2" column="17">
      <widget class="QLineEdit" name="lineEdit_35">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>v_a</string>
       </property>
      </widget>
     </item>
     <item row="1" column="10">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>q_b</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>i</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>vl_ca</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>i_b</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_45">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="18">
      <widget class="QLabel" name="label_40">
       <property name="text">
        <string>net_kvarh</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QLineEdit" name="lineEdit_20">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>p_c</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_45">
       <property name="text">
        <string>f</string>
       </property>
      </widget>
     </item>
     <item row="1" column="11">
      <widget class="QLineEdit" name="lineEdit_22">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="19">
      <widget class="QLineEdit" name="lineEdit_40">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="17">
      <widget class="QLineEdit" name="lineEdit_36">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>p</string>
       </property>
      </widget>
     </item>
     <item row="1" column="12">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>s_b</string>
       </property>
      </widget>
     </item>
     <item row="0" column="14">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>pf_a</string>
       </property>
      </widget>
     </item>
     <item row="1" column="15">
      <widget class="QLineEdit" name="lineEdit_30">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>i_c</string>
       </property>
      </widget>
     </item>
     <item row="1" column="13">
      <widget class="QLineEdit" name="lineEdit_26">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="12">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>s_c</string>
       </property>
      </widget>
     </item>
     <item row="3" column="12">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item row="3" column="14">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>pf</string>
       </property>
      </widget>
     </item>
     <item row="0" column="13">
      <widget class="QLineEdit" name="lineEdit_25">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="11">
      <widget class="QLineEdit" name="lineEdit_23">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="17">
      <widget class="QLineEdit" name="lineEdit_33">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="19">
      <widget class="QLineEdit" name="lineEdit_39">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>p_a</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="15">
      <widget class="QLineEdit" name="lineEdit_29">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QLineEdit" name="lineEdit_18">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QLineEdit" name="lineEdit_17">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="20">
      <widget class="QLabel" name="label_44">
       <property name="text">
        <string>net_kvah</string>
       </property>
      </widget>
     </item>
     <item row="1" column="20">
      <widget class="QLabel" name="label_42">
       <property name="text">
        <string>exp_kvah</string>
       </property>
      </widget>
     </item>
     <item row="2" column="10">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>q_c</string>
       </property>
      </widget>
     </item>
     <item row="3" column="13">
      <widget class="QLineEdit" name="lineEdit_28">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>i_a</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="13">
      <widget class="QLineEdit" name="lineEdit_27">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>vl_ab</string>
       </property>
      </widget>
     </item>
     <item row="0" column="18">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>imp_kvarh</string>
       </property>
      </widget>
     </item>
     <item row="0" column="19">
      <widget class="QLineEdit" name="lineEdit_37">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="15">
      <widget class="QLineEdit" name="lineEdit_32">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="14">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>pf_b</string>
       </property>
      </widget>
     </item>
     <item row="2" column="14">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>pf_c</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>v_c</string>
       </property>
      </widget>
     </item>
     <item row="2" column="15">
      <widget class="QLineEdit" name="lineEdit_31">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>p_b</string>
       </property>
      </widget>
     </item>
     <item row="0" column="12">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>s_a</string>
       </property>
      </widget>
     </item>
     <item row="2" column="18">
      <widget class="QLabel" name="label_39">
       <property name="text">
        <string>tot_kvarh</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>vl_bc</string>
       </property>
      </widget>
     </item>
     <item row="1" column="18">
      <widget class="QLabel" name="label_38">
       <property name="text">
        <string>exp_kvarh</string>
       </property>
      </widget>
     </item>
     <item row="1" column="21">
      <widget class="QLineEdit" name="lineEdit_42">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLineEdit" name="lineEdit_14">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>q_a</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="21">
      <widget class="QLineEdit" name="lineEdit_41">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="19">
      <widget class="QLineEdit" name="lineEdit_38">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="10">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>q</string>
       </property>
      </widget>
     </item>
     <item row="2" column="21">
      <widget class="QLineEdit" name="lineEdit_43">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="16">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>net_kwh</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QLineEdit" name="lineEdit_16">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QLineEdit" name="lineEdit_19">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="11">
      <widget class="QLineEdit" name="lineEdit_24">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>v</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>v_b</string>
       </property>
      </widget>
     </item>
     <item row="2" column="20">
      <widget class="QLabel" name="label_43">
       <property name="text">
        <string>tot_kvah</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>vl</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="16">
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>exp_kwh</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="17">
      <widget class="QLineEdit" name="lineEdit_34">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="16">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>imp_kwh</string>
       </property>
      </widget>
     </item>
     <item row="3" column="21">
      <widget class="QLineEdit" name="lineEdit_44">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MeterLink</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MeterLink</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
